dashboard/test/models/course_offering_test.rb

```diff
@@ -759,6 +759,20 @@ class CourseOfferingTest < ActiveSupport::TestCase
       new_course_offering.attributes.except('id', 'created_at', 'updated_at')
   end
 
+  test "can seed correct self paced pl id based on self paced pl key" do
+    course_offering = create :course_offering, key: 'course-offering-1', self_paced_pl_course_offering_id: 390
+    self_paced_pl_course = create :course_offering, key: 'self-paced-test-course'
+    serialization = course_offering.serialize
+    serialization[:self_paced_pl_course_offering_key] = "self-paced-test-course"
+
+    File.stubs(:read).returns(serialization.to_json)
+
+    new_course_offering_key = CourseOffering.seed_record("config/course_offerings/course-offering-1.json")
+    new_course_offering = CourseOffering.find_by(key: new_course_offering_key)
```

Comment:
it seems like there is a bit too much going on here and I think you can get rid of `new_course_offering_key` and do it like this:
```suggestion
    new_course_offering = CourseOffering.find_by(key: course_offering.key)
```
this is better because you are isolating the test to the part of the seed behavior where it updates the existing record, keeping the key return value out of it